libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFNetworkTests.m

```diff
@@ -0,0 +1,130 @@
+//
+//  SFNetworkTests.m
+//  SalesforceSDKCoreTests
+//
+//  Created by Brianna Birman on 8/1/19.
+//  Copyright © 2019 salesforce.com. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "SalesforceSDKCore/SFNetwork.h"
+
+@interface SFNetworkTests : XCTestCase <SFNetworkSessionManaging>
+
+@end
+
+#pragma clang diagnostic push
```

Comment:
Is this to hide protocol warnings?

Reply:
Yup, it's complaining about not implementing the two deprecated methods but putting them in gives a deprecated warning anyway